.github/workflows/test_frameworks.yml

```
@@ -238,17 +238,15 @@ jobs:
       - uses: actions/checkout@v2
         with:
           repository: encode/starlette
-          ref: 0.14.2
+          ref: 0.17.1
           path: starlette
       - name: Install ddtrace
         run: pip install ../ddtrace
       - name: Install dependencies
-        run: |
-          scripts/install
-          pip install graphene==2.1.9
+        run: scripts/install
       #Parameters for keyword expression skip 3 failing tests that are expected due to asserting on headers. The errors are because our context propagation headers are being added
```

```suggestion
      #Parameters for keyword expression skip 4 failing tests that are expected due to asserting on headers. The errors are because our context propagation headers are being added
```

Is that new test also failing due to asserting on headers?

---

I looked into the starlette ci and I don't think the run the test suite: [starlette](https://github.com/encode/starlette/blob/master/.github/workflows/publish.yml). 

I ran the job below using `act` and noticed that this test `test_staticfiles_with_invalid_dir_permissions_returns_401` fails even when ddtrace is not enabled: 

```
publish:
    name: "starlette tests"
    runs-on: "ubuntu-latest"
    defaults:
      run:
        working-directory: starlette
    steps:
      - uses: "actions/setup-python@v2"
        with:
          python-version: 3.7
      - uses: actions/checkout@v2
        with:
          repository: encode/starlette
          ref: 0.17.1
          path: starlette
      - name: "Install dependencies"
        run: "scripts/install"
      - name: Run tests
        run: pytest tests
```

This test was introduced about a month ago so I think it was never properly tested. I think it's safe to skip it for now.

---

👍🏻 safe to ignore then in our test cases.

Just make sure we have a good comment describing why we are skipping it.

---

We can either test against 0.17.1 and skip this test or just test against 0.17.0 (tests in v0.17.0 pass without any issues)